packages/sirius-web/backend/sirius-web-infrastructure/src/main/java/org/eclipse/sirius/web/infrastructure/elasticsearch/services/EditingContextIndexingService.java

```diff
@@ -101,10 +101,19 @@ public void index(IEditingContext editingContext) {
     private void clearIndex(String projectId) {
         this.optionalElasticSearchClient.ifPresent(elasticSearchClient -> {
             try {
-                elasticSearchClient.indices().delete(deleteIndexRequest -> deleteIndexRequest.index(ProjectIndexLifecycleManager.PROJECT_INDEX_NAME_PREFIX + projectId));
+                elasticSearchClient.indices().delete(deleteIndexRequest -> deleteIndexRequest
+                        .index(ProjectIndexLifecycleManager.PROJECT_INDEX_NAME_PREFIX + projectId)
+                        // The index may not exist yet. This can be the case if ProjectIndicesInitializer is still
+                        // running and hasn't processed the project yet.
+                        .ignoreUnavailable(true));
```

Comment:
This is the case in the tests: the `ProjectIndicesInitializer` does not see any project when it is executed.
When the application starts the SQL scripts to populate the databases haven't been executed yet (they are executed before each test), so there is nothing to index, and no indexes are created.

If later a test changes an editing context the `EditingContextIndexingService` will try to delete the index, but it doesn't exist.